### DISCUSSION TITLE
Can someone tell me how I can solve this problem?

### DISCUSSION THREAD

**Discussion by Luisillo0117** [PlayCover/PlayCover#1404] [Show and tell]
<img width="1470" alt="Captura de pantalla 2024-02-24 a las 14 11 57" src="https://github.com/PlayCover/PlayCover/assets/161163677/cc9777c5-7100-4603-8fd8-fc3e7f23ccaf">

Any idea why I get that error?
I already did the procedure to install play cover but this appears when I run Genshin Impact

**Comment by TheMoonThatRises** [comment]
Try doing the steps suggested in this issue: #1429.
